respy/parallelization.py

```diff
@@ -68,6 +68,47 @@ def wrapper_parallelize_across_dense_dimensions(*args, **kwargs):
         return decorator_parallelize_across_dense_dimensions
 
 
+def combine_and_split_interpolation(func):
```

Comment:
I like the way the function is written but I was wondering wehther it could make sense to run these regressions separately once the substates become to dissinmilair?

Reply:
I agree with you. I also thought multiple regressions for each dense dimensions might offer a better model fit and, thus, better interpolations. But, this would be a different interpolation method and we want to be as close to KW94 as possible. And this was simply designed for simpler state spaces, although Behrman (2016) and maybe some others use the same method for more complex examples.

Reply:
Ok makes sense!
Maybe something for the future if anyone wants to optimize in this dimension!